app/code/community/Bolt/Boltpay/controllers/ApiController.php

```diff
@@ -138,7 +138,7 @@ public function hookAction()
                         )
                     )
                 );
-                $this->getResponse()->setHttpResponseCode(200);
+                $this->getResponse()->setHttpResponseCode(200)->setBody(json_encode(array('status' => 'success')));
```

Comment:
Maybe better using the already exists function `sendResponse`, how do you think?